app/models/service_template_ansible_playbook.rb

```diff
@@ -51,16 +79,21 @@ def self.prepare_job_template_and_dialog(action, service_name, description, conf
   private_class_method :prepare_job_template_and_dialog
 
   def self.create_job_template(name, description, info)
-    playbook = ManageIQ::Providers::AnsibleTower::ConfigurationManager::Playbook.find(info[:playbook_id])
-    # tower = playbook.manager
-
-    # params = {
-    #   :name         => name,
-    #   :description  => description || '',
-    #   :extra_vars   => info[:variables] || {},
-    #   :inventory_id => playbook.inventory_root_group,
-    # }
-    # tower.class.create_in_provider(tower, params)
+    playbook = ManageIQ::Providers::AnsibleTower::AutomationManager::Playbook.find(info[:playbook_id])
+    tower = playbook.manager
+
+    params = {
+      :name                     => name,
+      :description              => description || '',
+      :project                  => playbook.configuration_script_source.manager_ref,
+      :playbook                 => playbook.name,
+      :inventory                => tower.inventory_root_groups.first.ems_ref,
+      :ask_variables_on_launch  => true,
```

Comment:
@bzwei - Regarding the empty `:network_credential`, `:cloud_credential` are there lookups we should be doing with them as well?  - Or that code isn't merged yet?